backend/hitas/tests/apis/helpers.py

```diff
@@ -20,13 +20,14 @@
 from openapi_core.validation.request.protocols import Request
 from openapi_core.validation.request.validators import APICallRequestValidator
 from openapi_core.validation.response import V30ResponseValidator
-from openapi_spec_validator.validation import openapi_spec_validator_proxy
+from openapi_spec_validator.validation import OpenAPIV30SpecValidator
 from rest_framework.response import Response
 from rest_framework.test import APIClient
 
 with open(f"{settings.BASE_DIR}/openapi.yaml", "r") as spec_file:
     openapi_spec_data = yaml.safe_load(spec_file)
-    openapi_spec_validator_proxy.validate(openapi_spec_data)
+    for err in OpenAPIV30SpecValidator(openapi_spec_data).iter_errors():
+        raise err
```

Comment:
Is this iterating necessary? `openapi_spec_validator` has an iteration test where they use the `validate()` method: https://github.com/python-openapi/openapi-spec-validator/blob/ca5447917451ea0cb3865238d6e547658a5cef04/tests/integration/validation/test_validators.py#L89.

Reply:
Changed this to `.validate()` because it does the same

```python
def validate(self) -> None:
    for err in self.iter_errors():
        raise err
```